gtsfm/loader/hilti_loader.py

```diff
@@ -54,29 +53,49 @@ def __init__(
         max_frame_lookahead: int = 10,
         step_size: int = 8,
         max_length: Optional[int] = None,
+        max_resolution: int = 1080,
     ) -> None:
-        super().__init__(max_resolution=1000)
+        """Initializes, loads calibration, constraints, and pose priors.
+
+        Args:
+            base_folder (str): top-level folder, expects calibration, images and lidar subfolders.
+            max_frame_lookahead (int, optional): creates relative pos priors up to this lookahead. Defaults to 10.
```

Comment:
Can you specify if 10 is in terms of images or in terms of rig indices? Also, should we make the default value very high to get loop closures from the constraints?

Reply:
I just commented your flag. I propose we remove it altogether